fuzzers/binary_only/fuzzbench_fork_qemu/src/fuzzer.rs

```diff
@@ -391,8 +393,9 @@ fn fuzz(
     // Remove target output (logs still survive)
     #[cfg(unix)]
     {
-        // dup2(null_fd, io::stdout().as_raw_fd())?;
-        // dup2(null_fd, io::stderr().as_raw_fd())?;
+        let null_fd = file_null.as_raw_fd();
```

Comment:
I wonder whyu this was commented out...

Reply:
@tokatoka looks like you did that, do you remember why?

Reply:
no probably i was debugging with this and then forgot to revert it back before merging to main